app/assets/javascripts/views/shared/social-tooltip.js

```diff
@@ -0,0 +1,21 @@
+(function (App) {
+
+  App.View.SocialTooltip = App.Component.Tooltip.extend({
+
+    content: JST['templates/shared/social-tooltip'],
+
+    initialize: function (options) {
+      this.refElem = options.refElem;
+      this.constructor.__super__.initialize.call(this, options);
```

Comment:
Maybe one good pattern would be to cache `this.constructor.__super__` as `this.super`?